ledger/participant-integration-api/src/main/scala/platform/store/dao/events/TransactionIndexing.scala

```diff
@@ -161,18 +129,11 @@ object TransactionIndexing {
         ),
         events = EventsInfo(
           events = events.result(),
-          archives = archived,
           stakeholders = stakeholders.result(),
           disclosure = disclosure.result(),
         ),
         contracts = ContractsInfo(
-          netCreates = netCreates,
-          netArchives = netArchives,
-          divulgedContracts = netDivulgedContracts,
-        ),
-        contractWitnesses = ContractWitnessesInfo(
-          netArchives = netArchives,
-          netVisibility = netVisibility,
+          divulgedContracts = netDivulged,
```

Comment:
Consider moving divulgedContracts into the `EventsInfo`, as we can think of a transaction containing a bunch of divulgence events alongside the create and exercise events.

Reply:
ha! even better so :)
done.